crypto/pem/pem_lib.c

```diff
@@ -362,7 +362,8 @@ int PEM_ASN1_write_bio(i2d_of_void *i2d, const char *name, BIO *bp,
             kstr = (unsigned char *)buf;
         }
         RAND_add(data, i, 0);   /* put in the RSA key. */
-        OPENSSL_assert(EVP_CIPHER_iv_length(enc) <= (int)sizeof(iv));
+        if (!ossl_assert(EVP_CIPHER_iv_length(enc) <= (int)sizeof(iv)))
+            goto err;
```

Comment:
This is effectively a compile-time condition. In sense that both terms are constant...